TextToSpeech/TextToSpeechImplementation.cpp

```diff
@@ -89,9 +106,51 @@ namespace Plugin {
         if(config.HasLabel("local_voices")) {
             JsonObject voices = config["local_voices"].Object();
             JsonObject::Iterator it = voices.Variants();
-            while(it.Next())
+
+            while(it.Next()) {
                 ttsConfig->m_others["voice_for_local_" + string(it.Label())] = it.Current().String();
+                expectedLanguageSet.insert(string(it.Label()));
+                expectedVoicesSet.insert(it.Current().String());
+            }
         }
+
+        #ifndef UNIT_TESTING
```

Comment:
Can you define these constants somewhere else (perhaps where you declare/define InputValidator) and keep this implementation simple?